pkgs/development/python-modules/deeplabcut/default.nix

```diff
@@ -0,0 +1,131 @@
+{ lib, buildPythonPackage, fetchFromGitHub, fetchzip, fetchurl
+, tensorflow
+, setuptools_scm
+, gzip
+, gnutar
+, certifi
+, chardet
+, click
+, easydict
+, h5py
+, imageio
+, imgaug
+, ipython
+, matplotlib
+, moviepy
+, numpy
+, opencv3
+, pandas
+, patsy
+, pillow
+, python
+, py
+, pyyaml
+, requests
+, ruamel_yaml
+, setuptools
+, scikitimage
+, scikitlearn
+, scipy
+, six
+, statsmodels
+, tables
+, tensorpack
+, tqdm
+, tkinter
+, wheel
+, wxPython_4_0
+, xvfb_run
+}:
+
+buildPythonPackage rec {
+  pname = "deeplabcut";
+  version = "2.1.6";
```

Comment:
Bump when upstream fixes the project creation bug where project creation fails without erroring.

Reply:
bumped!